src/ControlSystem/ControlErrors/Size.hpp

```diff
@@ -44,7 +46,34 @@ struct Distorted;
 }  // namespace Frame
 /// \endcond
 
-namespace control_system::ControlErrors {
+namespace control_system {
+namespace size {
+/*!
+ * \brief Function that computes the control error for
+ * `control_system::size::States::DeltaR`.
+ *
+ * This is helpful to have calculated separately because other control errors
+ * may make use of this quantity. This is Q in Eq. 96 in \cite Hemberger2012jz.
```

Comment:
(optional, if you agree my wording is clearer) The equation for the control error is given in Eq. (96) of \cite Hemberger2012jz.

Reply:
@geoffrey4444 squashed this in